packages/orderbook/src/seaport/seaport.ts

```diff
@@ -327,6 +433,34 @@ export class Seaport {
     );
   }
 
+  private async getTypedDataFromBulkOrderComponents(
+    orderComponents: OrderComponents[],
+    numberOfOrders: number,
+  ): Promise<SignableAction['message']> {
+    const { chainId } = await this.provider.getNetwork();
+
+    const domainData = {
+      name: SEAPORT_CONTRACT_NAME,
+      version: SEAPORT_CONTRACT_VERSION_V1_5,
```

Comment:
Is this something we want to put in to a config/env? Especially now that we use it in multiple places, could be easy to forget if we move to 1.6

Reply:
Those constant values are all imported from seaport/constants.ts. If we move versions updating there will modify all the places we create this message